src/pudl/metadata/resources.py

```diff
@@ -198,7 +198,7 @@
             ],
             # Null values in the generator_id field. See:
             # https://github.com/catalyst-cooperative/pudl/issues/1208
-            # "primary_key": ["plant_id_eia", "generator_id", "report_date"],
+            "primary_key": ["plant_id_eia", "generator_id", "report_date"],
```

Comment:
What years were the bad IDs showing up in? Were you able to run the ETL with this PK constraint enabled and process the data covering that year?

Reply:
I was able to run the ETL for all years with foreign key constraints ignored. I received a harvesting consistency error when running just 2008 and 2009 (the years with the missing ids). This is expected, right?

Reply:
Yeah, consistency errors are extremely likely with 2 years.